Formula/fortio.rb

```diff
@@ -11,29 +11,34 @@ def install
     ENV["XC_OS"] = "darwin"
     ENV["XC_ARCH"] = "amd64"
     ENV["GOPATH"] = buildpath
-    ENV["CGO_ENABLED"] = "0"
-    contents = Dir["{*,.git,.gitignore}"]
-    (buildpath/"src/istio.io/fortio").install contents
+
+    (buildpath/"src/istio.io/fortio").install buildpath.children
     cd "src/istio.io/fortio" do
       system "dep", "ensure"
-      date = `date +"%Y-%m-%d %H:%M"`.strip
+      date = Utils.popen_read("date", "+%Y-%m-%d %H:%M").strip
       link_flags = "-s -X istio.io/fortio/ui.resourcesDir=#{lib} "\
         "-X istio.io/fortio/version.tag=v#{version} "\
         "-X \"istio.io/fortio/version.buildInfo=#{date}\""
-      system "go", "build", "-a", "-ldflags", link_flags.to_s
-      bin.install "fortio"
+      system "go", "build", "-a", "-o", "#{bin}/fortio", "-ldflags", link_flags.to_s
       lib.install Dir["ui/static", "ui/templates"]
     end
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/fortio version -s")
+
+    require "socket"
+    server = TCPServer.new(0)
+    port = server.addr[1]
+    server.close
     fortio_srv = fork do
-      exec "#{bin}/fortio server -http-port 8085"
+      exec "#{bin}/fortio server -http-port #{port}"
```

Comment:
should be
```
fortio server --quiet --redirect-port "disabled" --grpc-port "disabled" -http-port 0 -ui-path ""
```
and get the URL from stdout $7 . (`|awk '{print $7}'`)

Reply:
for instance
```
docker run istio/fortio:0.9.0 server --quiet --redirect-port "disabled" --grpc-port "disabled" -http-port 0 -ui-path "" 2> /dev/null | awk '{print $7}'
[::]:36593
```
I'm fixing the stderr spurious message for grpc disable btw, and will make that that --quiet only prints the url

Reply:
Can we have this change in 0.9.1?

Reply:
yes sure if this passes as is it can wait